azure-container-apps/ADVANCED.md

```diff
@@ -0,0 +1,321 @@
+2. Click **Edit and deploy**.
+3. Select the checkbox next to your **op-scim-bridge** container, then choose **Edit**.
+4. Add a new **Environment variable**, using the **name** of `OP_DEBUG`, the **source** of **Manual entry** and enter `1` for the **value**
+5. Click **Save**, then click **Create**.
+
+This may be useful for troubleshooting, or when contacting 1Password Support.
+
+### Trace logs
+
+Set the environment variable `OP_TRACE` to `1` to enable trace level debug output in the logs:
+
+```bash
+az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge --set-env-vars OP_TRACE=1
+```
+
+To udpate within the Azure Portal: 
+
+1. Within Container App from the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), select **Containers** from the sidebar.
+2. Click **Edit and deploy**.
+3. Select the checkbox next to your **op-scim-bridge** container, then choose **Edit**.
+4. Add a new **Environment variable**, using the **name** of `OP_TRACE`, the **source** of **Manual entry** and enter `1` for the **value**
+5. Click **Save**, then click **Create**.
+
+This may be useful for troubleshooting issues.
+
+## Get help
+
+### Region support
+
+When you create or deploy the Container App Environment, Azure may present an error that the region isn't supported. You can review Azure documentation to make sure the region you selected supports [Azure Container Apps](https://azure.microsoft.com/explore/global-infrastructure/products-by-region/?regions=all&products=container-apps).
+
+### Container App Operational Insights error
+
+Some customers receive an error when creating the Azure Container App Environment stating that the subscription is not registered for Microsoft.OperationalInsights resource provider. This error occurs if the Operational Insights which is used by Log Analytics workspace for Container Apps has never been enabled within the subscription. Run the command shown in the error: 
+```
+az provider register -n Microsoft.OperationalInsights --wait
+```
+Once the command is completed, you will need to re-run the [command to create the Container App Environment](https://support.1password.com/scim-deploy-azure/#24-create-the-container-app-environment) again, so the command completes successfully with the Log Analystics workspace. 
+
+### Container App working with multiple subscriptions
+
+If your are using an existing resource group or have multiple subsriptions within your Azure Cloud environment, you may receive errors stating that the subscription or the resource group can not be found. Use the following command to set your subscription within your Shell. Alternatively you can add the `--subscription <subsciptionIDorName>` to every `az` command.
```

Comment:
```suggestion
If you're using an existing resource group or have multiple subscriptions within your Azure Cloud environment, you may see errors stating that the subscription or the resource group cannot be found. Use the following command to set your subscription within your Shell. Alternatively you can add the `--subscription <subsciptionIDorName>` to every `az` command.
```

Also, is this the same command for Bash and PowerShell?

Reply:
yes, this would be the same for both shells.